doc/architecture/computing-services.md

```diff
@@ -0,0 +1,105 @@
+
+Table of Contents
+=================
+
+   * [Table of Contents](#table-of-contents)
+   * [Motivation](#motivation)
+   * [Architecture](#architecture)
+      * [Enabling Publisher Services (Brizo)](#enabling-publisher-services-brizo)
+      * [Responsibilities](#responsibilities)
+      * [Initial solution](#initial-solution)
+      * [Next steps](#next-steps)
+
+
+---
+
+This page describes the highlights of the integration of additional computing services.
+
+# Motivation
+
+The most basic scenario of a Publisher, is to provide access to the Assets this Publisher owns or manage. In addition to this, having this data ecosystem in place,
+Publishers could offer other related services in top of this. Some possibles scenarios are:
+
+* The Publishers offer as a service the possibility of execute some computation on top of their data. This has some benefits:
+  - The data never leaves the Publisher enclave
+  - It's not necessary to move the data, the algorithm moves to the data
+  - Regulation
+
+* Storage services for new derived assets
+
+# Architecture
+
+## Enabling Publisher Services (Brizo)
+
+The direct interaction with the infrastructure where the data is, requires the execution of a component handled by Publishers.
+This component will be in charge of interact with the users, and manage the basics of own Publisher infrastructure to provide the additional services.
+This business logic supporting the some additional Publisher capabilities, is responsibility of a new technical component.
+
+The main & new key component introduced to support new Publisher services is named **Brizo**.
+
+_"Brizo is an ancient Greek goddess who was known as the protector of mariners, sailors, and fishermen. She was worshipped primarily by the women of Delos, who set out food offerings in small boats. Brizo was also known as a prophet specializing in the interpretation of dreams."_
+
+In the "Ocean ecosystem", Brizo is the technical component executed by the **Publishers** allowing to them to provide extended data services.
+Brizo, as part of the Publisher ecosystem, includes the credentials to interact with the infrastructure (initially cloud, but could be on-premise).
+Because of those credentials, the execution of Brizo **SHOULD NOT** be delegated to a third-party.
+
+![Brizo High-Level Architecture](img/brizo-hl-arch.png)
+
+
+## Responsibilities
+
+The main responsibilities of the software are:
+
+* Expose a HTTP api allowing to execute some extended data services (compute and/or storage)
+* Authorize the user on-chain using the proper Service Agreement. It allows to validate that user requesting the service is allowed to use that service.
```

Comment:
Maybe link "Service Agreement" to some page explaining what that means in Ocean.

Reply:
Good point. The SLA section is still in definition/review process. As soon as we get this published we can add a link to this page.